internal/display/test.go

```diff
@@ -0,0 +1,54 @@
+package display
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/auth0/go-auth0/management"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/auth0/auth0-cli/internal/auth/authutil"
+)
+
+type userInfoAndTokens struct {
+	UserInfo *authutil.UserInfo      `json:"user_info"`
+	Tokens   *authutil.TokenResponse `json:"tokens"`
+}
+
+func (r *Renderer) TestLogin(u *authutil.UserInfo, t *authutil.TokenResponse) {
+	r.Heading("/userinfo")
+
+	data := &userInfoAndTokens{UserInfo: u, Tokens: t}
+
+	r.JSONResult(data)
+}
+
+func (r *Renderer) TestToken(client *management.Client, t *authutil.TokenResponse) {
+	r.Heading(fmt.Sprintf("token for %s", ansi.Bold(client.GetName())))
+
+	switch r.Format {
+	case OutputFormatJSON:
+		r.JSONResult(t)
+	default:
+		if t.TokenType != "" {
+			r.Output("  TOKEN    TYPE   " + t.TokenType)
```

Comment:
Is this in line with the way we present tabular data in other commands?

Reply:
Unfortunately I opted to remove showing this as tabular data due to the length of the access tokens. If displayed in auto sizing columns it will make cells like the token type really big unnecessarily to be the same size as the access token cell as it's a table.